src/index.js

```diff
@@ -27,98 +27,101 @@ export function h(type, props) {
     : type(props || {}, children)
 }
 
-export function app(props, container) {
+export function app(model, view, container) {
```

Comment:
@JorgeBucaran we should add a trigger warning for folks who have PTSD from their MVC experiences

Reply:
@okwolf Haha!